eShop/Catalog/Catalog.Host/Controllers/CatalogBffController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Catalog.Host.Data.Entity;
+using Catalog.Host.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Catalog.Host.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class CatalogBffController : ControllerBase
+{
+    private readonly ICatalogBffRepository _catalogBffRepository;
+    private readonly ICatalogTypeRepository _catalogTypeRepository;
+    private readonly ICatalogBrandRepository _catalogBrandRepository;
+    private readonly ICatalogItemRepository _catalogItemRepository;
+    public CatalogBffController(
```

Comment:
Why this Controller contains all Get methods? Why don't we move them into their own Controllers class?